kahuna/public/js/preview/image.js

```diff
@@ -88,12 +88,11 @@ image.controller('uiPreviewImageCtrl', [
         'dead child',
         'child casualty',
         'sensitive material',
-        'injured',
-        'wounded',
-        'killed',
```

Comment:
these are too broad, and we don't want users to miss images and stop using the feature